### PR TITLE
env vars and vault for sensitive

### PR DESCRIPTION
**Pull request by bartlabno**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: infrastructure/aat.tfvars [hunk 1]
```diff
@@ -2,4 +2,5 @@ root_logging_level = "DEBUG"
 log_level_spring_web = "DEBUG"
 log_level_sscs = "DEBUG"
 management_security_enabled = "false"
-idam_oauth2_redirect_url = "https://sscs-case-loader-aat.service.core-compute-aat.internal"
```
Comment (iguannago) [inline infrastructure/aat.tfvars]: @bartlabno , we have you put this variable? It's needed to connect to CCD via IdAM.

File: infrastructure/main.tf [hunk 2]
```diff
@@ -47,22 +57,22 @@ module "sscs-case-loader" {
...
+    IDAM.S2S-AUTH = "${data.vault_generic_secret.idam_s2s_api.data["value"]}"
     IDAM.S2S-AUTH.MICROSERVICE = "${var.idam_s2s_auth_microservice}"
 
     IDAM_SSCS_SYSTEMUPDATE_USER = "${data.vault_generic_secret.idam_sscs_systemupdate_user.data["value"]}"
     IDAM_SSCS_SYSTEMUPDATE_PASSWORD = "${data.vault_generic_secret.idam_sscs_systemupdate_password.data["value"]}"
 
     IDAM_OAUTH2_CLIENT_ID = "${var.idam_oauth2_client_id}"
     IDAM_OAUTH2_CLIENT_SECRET = "${data.vault_generic_secret.idam_oauth2_client_secret.data["value"]}"
-    IDAM_OAUTH2_REDIRECT_URL = "${var.idam_oauth2_redirect_url}"
+    IDAM_OAUTH2_REDIRECT_URL = "https://sscs-case-loader-${var.env}.service.${local.asename}.internal"
```
Comment (iguannago) [inline infrastructure/main.tf]: there seems to be a type here: asename

File: infrastructure/aat.tfvars (hunk 1 above)
Comment (bartlabno) [reply infrastructure/aat.tfvars]: There is no need for this variable as it is replaced by: https://github.com/hmcts/sscs-case-loader/pull/53/files#diff-1031eeab417cd8afe77e7e005717d38dR71

[the remark this answers is not in the document]
Comment (bartlabno) [reply infrastructure/aat.tfvars]: There is no need for this variable as it is replaced by: https://github.com/hmcts/sscs-case-loader/pull/53/files#diff-1031eeab417cd8afe77e7e005717d38dR71

**Pull request merged**